src/screens/runPlan/PlanItemList.tsx

```diff
@@ -37,11 +43,27 @@ export class PlanItemList extends React.PureComponent<Props, State> {
     this.setState({ planItems });
   };
 
+  handleStudentChange = (documentSnapshot: RNFirebase.firestore.DocumentSnapshot) => {
+    if(documentSnapshot.exists){
+      this.setState({ student: documentSnapshot.data() });
+    }
+  }
+
+  completedPlanItemCounter() {
+    let countCompletedPlanItems = 0;
+    for (let planItem of this.state.planItems){
+      if(planItem.completed == true)
+        countCompletedPlanItems++;
+    }
+    return countCompletedPlanItems;
```

Comment:
What do you think about using reduce function on planItems here? I think it could be more "idiomatic JS styled" ;) Like e.g. in this example: https://www.freecodecamp.org/news/reduce-f47a7da511a9/

Reply:
ok I'll use it.